operations/helm/tests/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: AGPL-3.0-only
+
+set -euo pipefail
+
+CHART_PATH=operations/helm/charts/mimir-distributed
+
+# Start from a clean slate
+rm -rf operations/helm/tests/*values-generated
```

Comment:
nit: the suffix this script adds is `-generated`, not `-values-generated`
```suggestion
rm -rf operations/helm/tests/*-generated
```